src/main/java/io/nats/client/impl/NatsObjectStore.java

```diff
@@ -159,6 +157,9 @@ public ObjectInfo put(ObjectMeta meta, InputStream inputStream) throws IOExcepti
 
             throw e;
         }
+        finally {
+            try { inputStream.close(); } catch (IOException ignore) {}
+        }
```

Comment:
Be a good citizen. Not sure what to do if the close fails, it's not really relevant at that point anyway.